iP/src/duke/Ui.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+import java.util.ArrayList;
+import java.util.Scanner;
+/**
+ * Responsible for UI such as getting user input and outputting message to the user
+ */
+public class Ui {
+
+    /**
+     * shows error message occured while loading data from text file
+     */
+    public void showLoadingError() {
+        System.out.println("Failed to load data");
+    }
+
+    /**
+     * show welcome message when app is started
```

Comment:
Javadocs:
- "show" should be "Shows"
- add a full stop after the function description